src/utils/validation.ts

```diff
@@ -0,0 +1,13 @@
+import { ethers } from 'ethers';
+
+export const isValidEthereumAddress = (address: string | undefined): boolean => {
+    if (!address) return false;
+  
+    try {
+      return ethers.utils.isAddress(address);
```

Comment:
Does `ethers.utils.isAddress` actually throw an error or does it just return `true` / `false`? https://docs.ethers.org/v6/api/address/#isAddress it looks like the return type is just a boolean...

Reply:
Still unresolved